apps/dotcom/src/components/ErrorPage/ErrorPage.tsx

```diff
@@ -19,9 +20,7 @@ export function ErrorPage({
 					<p>{messages.para1}</p>
 					{messages.para2 && <p>{messages.para2}</p>}
 				</div>
-				<Link to={'/'}>
-					<a>Take me home.</a>
-				</Link>
+				<Link to={'/'}>{isInIframe() ? 'Open tldraw.' : 'Back to tldraw.'}</Link>
```

Comment:
this probably wants `target="_blank"` when `isInIframe` is true

Reply:
nice catch - thx!